cylc/flow/ws_data_mgr.py

```diff
@@ -834,8 +968,47 @@ def update_dynamic_elements(self, updated_nodes=None):
             return
         self.update_task_proxies(updated_nodes)
         self.update_family_proxies(set(str(t.point) for t in updated_nodes))
```

Comment:
Not in this change, but `update_family_proxies` doc specify a `list`. Probably harmless.